packages/backend-core/src/db/utils.ts

```diff
@@ -1,47 +1,26 @@
-const { newid } = require("../hashing")
-const Replication = require("./Replication")
-const { DEFAULT_TENANT_ID, Configs } = require("../constants")
-const env = require("../environment")
-const {
-  StaticDatabases,
-  SEPARATOR,
-  DocumentTypes,
-  APP_PREFIX,
-  APP_DEV,
-} = require("./constants")
-const { getTenantId, getGlobalDBName } = require("../tenancy")
-const fetch = require("node-fetch")
-const { doWithDB, allDbs } = require("./index")
-const { getCouchUrl } = require("./pouch")
-const { getAppMetadata } = require("../cache/appMetadata")
-const { checkSlashesInUrl } = require("../helpers")
-const {
-  isDevApp,
-  isProdAppID,
-  isDevAppID,
-  getDevelopmentAppID,
-  getProdAppID,
-} = require("./conversions")
+import { newid } from "../hashing"
+import { DEFAULT_TENANT_ID, Configs } from "../constants"
+import * as env from "../environment"
+import { SEPARATOR, DocumentTypes } from "./constants"
+import { getTenantId, getGlobalDBName } from "../tenancy"
+import fetch from "node-fetch"
+import { doWithDB, allDbs } from "./index"
+import { getCouchUrl } from "./pouch"
+import { getAppMetadata } from "../cache/appMetadata"
+import { checkSlashesInUrl } from "../helpers"
+import { isDevApp, isDevAppID } from "./conversions"
 
 const UNICODE_MAX = "\ufff0"
 
-exports.ViewNames = {
+export const ViewNames = {
   USER_BY_EMAIL: "by_email",
   BY_API_KEY: "by_api_key",
   USER_BY_BUILDERS: "by_builders",
 }
 
-exports.StaticDatabases = StaticDatabases
-
-exports.DocumentTypes = DocumentTypes
-exports.APP_PREFIX = APP_PREFIX
-exports.APP_DEV = exports.APP_DEV_PREFIX = APP_DEV
```

Comment:
@mike12345567 not sure if I've converted this properly, I don't follow what the double `=` is doing. Below I've just re-exported conversions and contants for handiness. Probably we move `db/utils.ts` (and associated  Replication, constants, conversions) into -> `db/utils` and introduce `db/utils/index.ts` which will export things at the `utils` module level

Reply:
The double equals is basically just equivalent to `a = b = 1` which means that `a = 1, b = 1`, essentially setting them all in one flow to the same value.

Reply:
ah great, thanks!